src/lib/utils.ts

```diff
@@ -449,6 +449,14 @@ export async function parseCommandLineArgs(args: string[], usage: string) {
     }
   }
 
+  // Parse host
+  let host = '127.0.0.1' // Default
```

Comment:
Given the existing code assumes `localhost`, it feels like that should remain the default here. I know https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-12#section-8.4.2 says what it says but it feels a bit strange to enforce in a dynamic client registration world (which arguably undermines a lot of the meaningful restrictions on clients).

Wdyt?

Reply:
Yeah I agree on keeping localhost the default for now - the decision should ultimately be up to the IDP what to allow or not allow anyway. updated in https://github.com/geelen/mcp-remote/pull/82/commits/c0f7c49d059d0247176080ff7015d308b783da56